pkg/api/v2/reporter/reporter.go

```diff
@@ -19,6 +22,11 @@ import (
 type Report struct {
 	Warnings []string
 	Errors   error
+
+	// Additional information about the current state of the resource.
+	Messages []string `protobuf:"bytes,3,opt,name=message,proto3" json:"message,omitempty"`
+	// The last time the status was updated.
+	UpdatedAt *timestamp.Timestamp `protobuf:"bytes,4,opt,name=updated_at,json=updatedAt,proto3" json:"updated_at,omitempty"`
```

Comment:
nit, let's remove the proto struct tags, since these are not protos. we can keep the json ones per https://gobyexample.com/json

Reply:
also i don't believe `UpdatedAt` is used anywhere

Reply:
I meant to ask you about that, is this something we want to support or should it be removed.

Reply:
Removed